src/components/learner-credit-management/LearnerCreditAllocationTable.jsx

```diff
@@ -34,35 +40,45 @@ const LearnerCreditAllocationTable = ({
       defaultColumnValues={{ Filter: TableTextFilter }}
       columns={[
         {
-          Header: 'Email Address',
-          accessor: 'userEmail',
+          Header: 'Date',
+          accessor: 'enrollmentDate',
           // eslint-disable-next-line react/prop-types, react/no-unstable-nested-components
-          Cell: ({ row }) => <EmailAddressTableCell row={row} enterpriseUUID={enterpriseUUID} />,
+          Cell: ({ row }) => dayjs(row.values.enrollmentDate).format('MMM D, YYYY'),
+          disableFilters: true,
         },
         {
-          Header: 'Course Name',
-          accessor: 'courseTitle',
+          Header: 'Description',
+          accessor: getDescriptionAccessor,
+          // eslint-disable-next-line react/prop-types, react/no-unstable-nested-components
+          Cell: ({ row }) => (
+            <>
+              <Avatar size="xs" />{' '}
+              <EmailAddressTableCell row={row} enterpriseUUID={enterpriseUUID} />
+              <div>
+                <Hyperlink
+                  // eslint-disable-next-line react/prop-types
+                  destination={`${getConfig().ENTERPRISE_LEARNER_PORTAL_URL}/${enterpriseSlug}/course/${row.original.courseKey}`}
+                  target="_blank"
+                >
+                  {/* eslint-disable-next-line react/prop-types */}
+                  {row.original.courseTitle}
+                </Hyperlink>
+              </div>
+            </>
+          ),
```

Comment:
IMO, either we should update the ESLint comment above to ensure we disable the rule for an entire block of code, not just the next line. That way, you wouldn't need to include multiple comments for `eslint-disable-next-line react/prop-types` within the same sub-component.

Alternatively, it may make sense to also abstract this into its own standalone component to avoid these ESLint issues altogether. 

Example:

```js
{
  Cell: DescriptionTableCell,
}
```